examples/c5g7_transient/rod_translation/materials_mg.py

```diff
@@ -0,0 +1,48 @@
+import openmc
```

Comment:
This file is almost identical to `density_change/materials_mg.py`. Consider removing it and using a symbolic link instead